pkg/container/qemu_runner.go

```diff
@@ -298,7 +298,8 @@ func (bw *qemu) Debug(ctx context.Context, cfg *Config, envOverride map[string]s
 
 	clog.FromContext(ctx).Debugf("running debug command: %v", args)
 
-	err = session.Shell()
+	cmd := shellquote.Join(args...)
+	err = session.Run(cmd)
```

Comment:
This doesn't look right.  Won't this run the command instead of creating a Shell?

Reply:
well the command is a shell (it's cd /home/build && /bin/sh)
it works well from my local testing, you have a shell resizable and with all the ctrl-c/d/a/z stuff working

Reply:
Got it.  I thought this was the command being debugged.